py3status/modules/net_rate.py

```diff
@@ -50,39 +65,36 @@ class Py3status:
     devfile = '/proc/net/dev'
     format = "{interface}: {total}"
     format_no_connection = ''
+    format_value = "{value:5.1f} {unit:>5s}"
```

Comment:
Seeing how #546 is in, should I rewrite the default using the new `min_length`? (this also applies to #530 and #541)